packages/app/src/features/NoteEditor/NoteVersions/NoteVersions.tsx

```
@@ -0,0 +1,351 @@
+	);
+
+	useEffect(() => {
+		setVersions(null);
+		updateVersionsList();
+	}, [noteId, updateVersionsList]);
+
+	// Refresh note versions by event
+	const eventBus = useEventBus();
+	useEffect(() => {
+		return eventBus.listen(WorkspaceEvents.NOTE_HISTORY_UPDATED, (noteId) => {
+			if (noteId !== noteId) return;
```

Biome (2.4.16)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)